pages/dkp/kommander/1.4/clusters/tunnel-cli/index.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: layout.pug
+beta: true
+navigationTitle: Attach a Kubernetes Cluster using a Tunnel
+title: Attach Cluster using tunnel
+menuWeight: 7
+excerpt: Using the CLI to attach a Kubernetes Cluster using a Tunnel
+---
+
+## Initial setup
+
+### Identify the management cluster endpoint
+
+Obtain the hostname and CA certificate for the management cluster:
+```shell
+hostname=$(kubectl get service -n kubeaddons traefik-kubeaddons -o jsonpath="{.status.loadBalancer.ingress[0].hostname}")
+b64ca_cert=$(kubectl get secret -n cert-manager kubernetes-root-ca -o=go-template='{{index .data "tls.crt"}}')
+```
+
+### Create a namespace
+
+Create a namespace on the management cluster for the tunnel gateway:
+```shell
+namespace=sample
+
+cat > namespace.yaml <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${namespace}
+EOF
+
+kubectl apply -f namespace.yaml
+```
+
+You can verify the namespace exists using:
+```shell
+kubectl get namespace ${namespace}
+```
+
+
+### Create a tunnel gateway
+
+Create a tunnel gateway on the management cluster to listen for tunnel agents on remote clusters:
+```shell
+name_cacert=kubetunnel-ca
+name_gateway=sample-tunnel-gateway
+
+cat > gateway.yaml <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: ${namespace}
+  name: ${name_cacert}
+data:
+  ca.crt:
+    ${b64ca_cert}
+---
+apiVersion: kubetunnel.d2iq.io/v1alpha1
+kind: TunnelGateway
+metadata:
+  namespace: ${namespace}
+  name: ${name_gateway}
+spec:
+  ingress:
```

Comment:
We should include the table here with the definition of the api types included in the kubetunnel repository. Perhaps we can start copying from there here, to continue in the future with direct PRs to this page, so we don't miss new api types.

Reply:
I meant this doc https://github.com/mesosphere/kubetunnel/blob/master/docs/api-types/index.md

Reply:
or maybe simply add a brief intro to the names and their purpose.

Reply:
Since the linked page is ready to insert as a standalone page, we can include it as a separate page now, and link to it here.

Reply:
sounds good to me.